src/lib/components/forms/tests/DocumentUpload.test.ts

```diff
@@ -30,7 +30,7 @@ describe("DocumentUpload form", () => {
   it("provides feedback when a file is too large", async () => {
     render(DocumentUploadForm);
     const dropElement = screen.getByText(
-      "Select, drag and drop, or paste files to upload",
+      "Select, paste or drag and drop files to upload",
```

Comment:
If previous suggestion is accepted, then fix:

```suggestion
      "Select, paste or drag-and-drop files to upload",
```